File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// VisitorFrequenciesLucerneCh SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class VisitorFrequenciesLucerneChMakeContext
{
    public static function call(array $ctxmap, ?VisitorFrequenciesLucerneChContext $basectx): VisitorFrequenciesLucerneChContext
    {
        return new VisitorFrequenciesLucerneChContext($ctxmap, $basectx);
    }
}

File: py/visitorfrequencieslucernech_sdk/utility/make_context.py
# VisitorFrequenciesLucerneCh SDK utility: make_context

from visitorfrequencieslucernech_sdk.core.context import VisitorFrequenciesLucerneChContext


def make_context_util(ctxmap, basectx):
    return VisitorFrequenciesLucerneChContext(ctxmap, basectx)
